python/guide/packages/gluon/index.rst

```diff
@@ -113,6 +113,12 @@ Advanced Topics
 
       A guide to implementing custom layers.
 
+   .. card::
```

Comment:
I think a beginner guide should not be in the Advanced section. Do you think it is too hard for it to be in the Getting Started section? Or, maybe it should nest under custom layers so you have a new index page for in a custom_layers folder with the beginner guide first.

Reply:
Moved this to the getting started section since it provides a good path for new users and includes some foundational info.